docs/developer-guide/developing.md

```diff
@@ -120,6 +120,35 @@ devServer: {
 
 * You have to run npm start to run mapstore client on port 8081, that is now connected to the local test back-end
 
+### Using local backend and setting up tomcat server
+* Download a tomcat standalone [here](https://mapstore.readthedocs.io/en/latest/developer-guide/requirements/) and extract to a folder of your choice
```

Comment:
```suggestion
If you prefer, or if you have some problems with `mvn jetty:run-war`, you can run MapStore back-end in a tomcat instance instead of using Jetty. 
To do so, you can :
* download a tomcat standalone [here](https://mapstore.readthedocs.io/en/latest/developer-guide/requirements/) and extract to a folder of your choice
```